.github/ISSUE_TEMPLATE/enhancement-request.yaml

```
@@ -0,0 +1,89 @@
+name: "Security docs enhancement request"
+description: Submit an issue to the security documentation team to request an improvement to the Elastic Security documentation.
+title: "[enhancement]: "
+labels: "enhancement"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hello! Use this form to submit an enhancement request to the [@elastic/security-docs](https://github.com/orgs/elastic/teams/security-docs) team. 
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Summarize your request here. What is the problem?
+      placeholder: |
+        What: We need documentation for X.
+        Why: We've received support requests that indicate customers are struggling with Y in versions A and B. 
+    validations:
+      required: true
+  - type: textarea
+    id: related
+    attributes:
+      label: Related links / assets
+      description: How can we learn more about the problem?
+      value: |
+        Please include each of the following, if applicable:
+        Subject matter expert:
```

Let's add an option for the relative docs link. 
```suggestion
        Doc URL: 
        Subject matter expert:
```